src/app/static/src/js/components/runWorkflow/runWorkflowProgress.vue

```diff
@@ -29,13 +29,36 @@
             <div class="row" v-if="workflowRunStatus" id="workflow-table">
                 <label class="form-label col">Reports</label>
                 <table class="table-bordered col-10">
-                    <tr v-for="report in workflowRunStatus.reports">
+                    <tr v-for="(report, index) in workflowRunStatus.reports" :key="report.key">
                         <td v-if="report.status === 'success'" class="p-2">
                             <a class="report-version-link" :href="reportVersionHref(report.name, report.version)">
                                 {{ report.name }}
                             </a>
                         </td>
                         <td v-else class="p-2">{{ report.name }}</td>
+                        <td v-if="anyParams">
+                            <span class="text-muted d-inline-block">Parameters</span>
```

Comment:
Alternatively, we could add `th` row to the top of the table (ie, add a header row for each column (Report name, Parameters, Status, Logs).